### PR TITLE
A few fixes for the JSON Sample Schema

### PR DESCRIPTION
**Pull request by ilsley**
Addresses issues #86 and #87.

### CODE REVIEW

File: schemas/json/1.0/sample.json [hunk 1]
```diff
@@ -108,13 +108,13 @@
...
                 "donor_sex" : {"type": "array", "minItems": 1, "maxItems": 1, "items": {"type": "string", "enum": ["Male", "Female", "Unknown", "Mixed"], "description": "'Male', 'Female', 'Unknown', or 'Mixed' for pooled samples."}},
                 "donor_ethnicity" : {"type": "array", "minItems": 1, "maxItems": 1, "items": {"type": "string", "description": "The ethnicity of the donor that provided the primary cell. NA if not available. If dealing with small/vulnerable populations consider identifiability issues."}}
             },
-            "required": ["donor_id", "donor_age", "donor_age_unit", "donor_life_stage", "donor_health_status_uri", "donor_health_status", "donor_sex", "donor_ethnicity"]
```
Comment (sitag) [inline schemas/json/1.0/sample.json]: I suspect this will cause problems for already submitted metadata

File: version_metadata/prevalidate.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def sample(self, js):
 
 		for subtype in js["allOf"]:
 			print(subtype)
-			biomaterial_type = subtype['if']['properties']['biomaterial_type']["const"]
+			biomaterial_type = subtype['if']['properties']['biomaterial_type']["const"][0]
```
Comment (sitag) [inline version_metadata/prevalidate.py]: cmn.uniq would be preferrable, but his is good for now.

File: schemas/json/1.0/sample.json (hunk 1 above)
Comment (ilsley) [reply schemas/json/1.0/sample.json]: Do you mean the switch to donor_health_status_ontology_uri? Does already submitted metadata comply with donor_health_status_uri as a required field?

**Comment by juettemann** [comment]
I will update my request. This solves the donor issue on both 1.0 and

**Pull request merged**